scraper.rb

```diff
@@ -82,18 +82,27 @@ def constituencies
 # ----------
 
 new_format_terms = {
-  '2016' => 'https://en.wikipedia.org/wiki/List_of_members_of_the_parliament_of_Iceland',
-  '2013' => 'https://en.wikipedia.org/wiki/List_of_members_of_the_parliament_of_Iceland,_2013%E2%80%9316',
+  '2016' => 'https://en.wikipedia.org/wiki/Template:MembersAlthing2016',
+  '2013' => 'https://en.wikipedia.org/wiki/Template:MembersAlthing2013',
 }
 
-new_format_terms.each do |term, url|
-  page = MembersPageWithAreaTable.new(response: Scraped::Request.new(url: url).response)
-  members = page.members.each { |m| m[:term] = term }
-  # puts members
-  puts "#{term}: #{members.count}"
-  ScraperWiki.save_sqlite(%i(name term), members)
+old_revisions = {
+  '2013' => 'https://en.wikipedia.org/w/index.php?title=Template:MembersAlthing2013&direction=prev&oldid=714280236',
+}
+
+def scrape_new_format_terms(terms)
+  terms.each do |term, url|
+    page = MembersPageWithAreaTable.new(response: Scraped::Request.new(url: url).response)
+    members = page.members.each { |m| m[:term] = term }
```

Comment:
I realise that this code pre-existed, but as you're now wrapping this up into a new method, it's probably worth bringing this in line with how we tend to do this now, as this idiom is a bit confusing.  This scraper as a whole is still quite a mess, and we don't need to fix it all up in one shot, but when working on a particular piece of good, it's good to bring it in line with our current approach where that's sensible, and here that looks quite simple to do.